tests/Driver/DropzoneUploadDriverTest.php

```diff
@@ -106,7 +106,7 @@ public function testUploadMonolithWithCallback()
     public function excludedPostParameterProvider()
     {
         return [
-            'dzuuid' => ['dzuuid'],
+            'dzuid' => ['dzuid'],
```

Comment:
You should not change the name of the parameter. https://github.com/coding-socks/laravel-chunk-uploader/issues/4#issuecomment-487366976